src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -177,7 +177,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.updateTheme(this.themeService.getColorTheme());
 		this.setFocusAndScroll();
 		this.cellModel.isEditMode = false;
-		this._htmlMarkdownConverter = new HTMLMarkdownConverter(this.notebookUri);
+		this._htmlMarkdownConverter = new HTMLMarkdownConverter(this.notebookUri, this._configurationService);
```

Comment:
I guess this is fine since we already have the configuration service - but generally for proper DI usage we should be creating objects using instantiation service 

`this._htmlMarkdownConverter = this._instantiationService.createInstance(HTMLMarkdownConverter, this.notebookUri)`